### DISCUSSION TITLE
ESP32S Wroome Heating

### DISCUSSION THREAD

**Discussion by Suparnpree** [Moddable-OpenSource/moddable#1335] [Q&A]
Hello everyone I am having a problem, whenever I connect the esp32 with my computer USB it just start heating so much. I have connected nothing to it I am just uploading hello world code. The red light is also ON on esp32. 
Please tell me how can I get that.

**Comment by phoddie** [comment]
I've never seen a board overheat running our [helloworld](https://github.com/Moddable-OpenSource/moddable/blob/public/examples/helloworld/main.js) (or any other example in the Moddable SDK). This doesn't seem likely to be a software problem. Do you have another board you can try?

**Reply by Suparnpree** [reply]
this is my new esp32 i have tried it 1st time

**Reply by chris-moddable** [reply]
Is this happening on a Moddable development board or another type of development board? As suggested above have you tried another board?
